config/initializers/health_monitor.rb

```diff
@@ -13,6 +13,10 @@
 
     config.path = :health
 
+    config.file_absence.configure do |file_config|
+      file_config.filename = "public/remove-from-nginx"
+    end
+
     config.error_callback = proc do |e|
       Rails.logger.error "Health check failed with: #{e.message}"
       Honeybadger.notify(e)
```

Comment:
In the example we don't report to Honeybadger if it's this specific check, so it doesn't get hammered

```suggestion
      Honeybadger.notify(e) unless e.is_a?(HealthMonitor::Providers::FileAbsenceException)
```